README.md

```diff
@@ -1 +1,86 @@
-# tv-maze
\ No newline at end of file
+# Tv maze
+
+Your site for watching any movie for free.
+
+## Built With
+
+- Major languages: HTML, CSS & Javascript
+- Technologies used: Linters, Git and Github, webpack and jest
+
+## Live Demo
+
+[https://john-kibirige.github.io/tv-maze/](https://john-kibirige.github.io/tv-maze/)
+
+## Video description
+
+[Video description](https://drive.google.com/file/d/1epM65U88fLm-noMRWbFB-hd_wzpA83UB/view?usp=sharing)
+
+## Getting Started
+
+### Prerequisites
+
+You should be having [Node.js](https://nodejs.org/en/) installed on your computer
+
+### Setup
+
+To get a local copy up and running follow these simple example steps.
+
+1. Navigate to a folder where you want the cloned file to appear
+
+2. Clone this repo in your terminal or git bash using the command
+
+   ```
+   git@github.com:John-Kibirige/tv-maze.git
+   ```
+
+3. run `npm install ` to install all the necessary dependencies including the linters
+
+4. Navigate to the folder called tv-maze/ and open this project using your editor
+
+### Run tests
+
+These were linter tests mainly
+
+- For css, run `npx hint .`
+- Html, run `npx stylelint "**/*.{css,scss}"`
+- Javascript `npx eslint .`
+
+- run , `npm run start` for setting up the local server
+- run, `npm run build` for building the project using webpack
+- run, `npm test` for seeing all the tests
+
+### Deployment
+
+## Authors
+
+👤 **Melashu Amari**
+
+- [Github](https://github.com/melashu)
+- [Twitter](https://twitter.com/meshu102)
+- [LinkedIn](https://twitter.com/home)
```

Comment:
- [ ] OPTIONAL Please, consider removing not related links to each author. Here is the LinkedIn link is linked to the home page 👍 You can remove it or fix the correct link 🥇